src/gui/mainwindow.h

```diff
@@ -187,7 +187,9 @@ private slots:
     void toolbarTextBeside();
     void toolbarTextUnder();
     void toolbarFollowSystem();
-#ifndef Q_OS_MAC
+#ifdef Q_OS_MAC
+    void on_actionCloseWindow_triggered();
```

Comment:
Maybe don't create this intermediate slot and just connect action to window `close()` slot directly?

Reply:
@glassez Just like with the previous comment, I'd expect other OS to use this, and it will not be a simple `close()` on say Linux or Windows. It will be kind of not very future-proof.